app/controllers/dish_controller.rb

```diff
@@ -11,5 +11,7 @@ def add_to_cycle
             Menu.add_dishes_to_cycle(params[:dish_date], [dish])
         end
         
+        # redirect to main page or show complete page
```

Comment:
Minor thing about this file, usually controllers are named with the plural of the model name, so this would be `DishesController` https://guides.rubyonrails.org/action_controller_overview.html#controller-naming-convention

Reply:
(Another comment also about changes that predate this PR, sorry)

It's preferable to stick with the rails conventions of `index`, `show`, `new`, `edit`, `update`, `create`, and `destroy` controller methods whenever possible so you can use more built in convenience methods. This looks like a `create` method to me, so it would be better to call it `create` unless it's doing something other than creating an object based on form input.